### PR TITLE
ChoiceGroup: fixed broken required style for label

### PR DESCRIPTION
**Pull request by fantua**
#### Pull request checklist

- [x] Addresses an existing issue: #1185
- [ ] Include a change request file if publishing 
- [x] Bugfix
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

Added use of Label component instead of own label implementation. Also removed component `className` prop from `Label`, logically looks like `className` need to be added only to a component container.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Image } from '../../Image';
-import { Icon } from '../../Icon';
+import { Label } from '../../Label';
+import { Icon, IIconProps } from '../../Icon';
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx]: Is this Icon/IIconProps necessary?

Comment (fantua) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx]: Yes, option can be an icon.

**Comment by dzearing** [comment]
@fantua Micah was saying you were importing something that wasn't being used (`IIconProps`), that's why build was failing. I've fixed it for you. Will try to merge.

**Comment by dzearing** [comment]
This has been merged already and published in 2.0.1.

**Pull request closed without merging**